src/components/science/ScienceGraphTable.jsx

```
@@ -0,0 +1,181 @@
+    createData("clear", 0, 0),
+    createData("purple", 0, 0),
+    createData("blue", 0, 0),
+    createData("red", 0, 0),
+  ]);
+  const [purpleAbsorbance, setPurpleAbsorbance] = useState([]);
+  const [blueAbsorbance, setBlueAbsorbance] = useState([]);
+  const [redAbsorbance, setRedAbsorbance] = useState([]);
+  const [frequency, setFrequency] = useState([]);
+  const [running, setRunning] = useState(false);
+  const [finished, setFinished] = useState(false);
+  const [stepIndex, setStepIndex] = useState(0);
```

ESLint

[error] 'stepIndex' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)